edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Service/EdgeHubCertificates.cs

```diff
@@ -19,10 +19,13 @@ public class EdgeHubCertificates
 
         public IList<X509Certificate2> CertificateChain { get; }
 
-        EdgeHubCertificates(X509Certificate2 serverCertificate, IList<X509Certificate2> certificateChain)
+        public IList<X509Certificate2> TrustBundle { get; }
+
+        EdgeHubCertificates(X509Certificate2 serverCertificate, IList<X509Certificate2> certificateChain, IList<X509Certificate2> trustBundle)
         {
             this.ServerCertificate = Preconditions.CheckNotNull(serverCertificate, nameof(serverCertificate));
             this.CertificateChain = Preconditions.CheckNotNull(certificateChain, nameof(certificateChain));
```

Comment:
How is the TrustBundle different from the CertificateChain?

Reply:
Trustbundle collection of unrelated root CAs
  **Example:**  FLEET 1 CA | FLEET 2 CA | ...

Certificate Chain is only the Edge Hub server certificate chain. 
  **Example Cert**: Edge Hub Server Cert | INT 2 | INT 1 |  Root CA
  In the example above, the chain would be INT 2 | INT 1 |  Root CA

Reply:
Isn't TrustBundle what is returned to other modules on the Workload API call, and what the modules use to validate the EdgeHub Server Certificate? Or is this something different? In which case, how would the TrustBundle above work?

Reply:
Yes that is right it is the same TrustBundle that is returned to all modules. All modules including Edge Hub would receive the same set of CA certificates. One among them would be the root of the Edge Hub server certificate.